deliver/lib/deliver/app_screenshot.rb

```diff
@@ -145,7 +196,10 @@ def self.calculate_screen_size(path)
         skip_landscape = true
       end
 
-      self.devices.each do |device_type, array|
+      # iMessage screenshots have same resolution as app screenshots so we need to distinguish them
+      devices = path_component.eql?("iMessage") ? device_messages : self.devices
```

Comment:
So the screenshots must be inside of an `iMessage` folder or contain it in the name? If so, we have to document this somewhere